### PR TITLE
Update WinUI to use HybridCRT

### PR DESCRIPTION
**Pull request by kmahone**
Following an approach from this change in [WindowsAppSDK - Use the Universal C Runtime](https://github.com/microsoft/WindowsAppSDK/pull/888) we switch WinUI over to use the HybridCRT.

This removes the dependency on VCLibs for Microsoft.UI.Xaml. VCLibs is still required for most UWP scenarios, but there are Xaml islands scenarios where it is helpful to not have to depend on other framework packages.

I have validates that apps build against this updated Microsoft.UI.Xaml package continue to pass WACK and continue to run correctly on non-desktop skus.

### CODE REVIEW

**Review by Scottj1s — APPROVED** [review]

**Comment by Scottj1s** [comment]
I assume the WACK questions were all resolved?

**Comment by kmahone** [comment]
> I assume the WACK questions were all resolved?

Yes. I ran WACK against a build with this change and did not hit any failures.

File: test/IXMPTestApp/TAEF/IXMPTestApp.TAEF.csproj [hunk 1]
```diff
@@ -10,6 +10,12 @@
   <PropertyGroup>
     <ProjectGuid>{74F24BC4-794D-4CB2-8420-80FF7FDACFE9}</ProjectGuid>
   </PropertyGroup>
+  <ItemGroup>
```
Comment (jevansaks) [inline test/IXMPTestApp/TAEF/IXMPTestApp.TAEF.csproj:13]: Unrelated change?

**Review by jevansaks — APPROVED** [review]
Awesome, thank you for doing this!
